.github/workflows/ci.yml

```diff
@@ -99,7 +104,10 @@ jobs:
   release:
     name: Release
     needs: build
-    if: startsWith(github.ref, 'refs/tags/v') || (github.event_name == 'workflow_dispatch' && github.event.inputs.publishSnapshot == 'true') # || (github.ref == 'refs/heads/main')
+    if:
```

Comment:
FYI I tested this out in a smaller repo: https://github.com/kubukoz/demos/actions/runs/5581376805/jobs/10199450921